docs/HOL_Deploying-Using-Puppet/README.md

```diff
@@ -310,7 +223,7 @@ flesh out the rest of the module properly.
     Let's edit the `init.pp` file of the `mrpapp` module (this is the entry point for the module):
 
     ```sh
-    sudo nano /etc/puppetlabs/puppet/environments/production/modules/mrpapp/manifests/init.pp
+    sudo nano /etc/puppetlabs/code/environments/production/modules/mrpapp/manifests/init.pp
     ```
 
```

Comment:
The link to the init.pp at the very last line of the document must be updated to point to the Microsoft repo